docs/source/cheat_sheet.rst

```diff
@@ -206,43 +217,45 @@ Classes
 .. code-block:: python
 
    class MyClass(object):
-
-       # For instance methods, omit `self`.
+       # For instance methods, omit type for "self"
        def my_method(self, num, str1):
            # type: (int, str) -> str
            return num * str1
 
-       # The __init__ method doesn't return anything, so it gets return
-       # type None just like any other method that doesn't return anything.
+       # The "__init__" method doesn't return anything, so it gets return
+       # type "None" just like any other method that doesn't return anything
        def __init__(self):
            # type: () -> None
            pass
 
-   # User-defined classes are written with just their own names.
-   x = MyClass() # type: MyClass
+   # User-defined classes are valid as types in annotations
+   x = MyClass()  # type: MyClass
 
 
-Other stuff
-***********
+Miscellaneous
+*************
 
 .. code-block:: python
 
    import sys
-   # typing.Match describes regex matches from the re module.
+   import re
    from typing import Match, AnyStr, IO
-   x = re.match(r'[0-9]+', "15") # type: Match[str]
 
-   # Use AnyStr for functions that should accept any kind of string
-   # without allowing different kinds of strings to mix.
+   # "typing.Match" describes regex matches from the re module
+   x = re.match(r'[0-9]+', "15")  # type: Match[str]
+
+   # You can use "AnyStr" to indicate that any string type will work,
+   # and the return type should match argument types
```

Comment:
I think we should link to the docs for AnyStr or something here. I don't think that it's obvious AnyStr is supposed to be a TypeVar and that you're supposed to use it at least twice in a given signature, but these are probably things a user ought to know.

Reply:
I decided to leave this section out since `AnyStr` is too hard to explain in a cheat sheet. Besides, most correct uses of `AnyStr` seem to be in library stubs.